etc/version_parser.ml

```diff
@@ -0,0 +1,17 @@
+
+let is_number x = try let _ = int_of_string x in true with _ -> false ;;
+
+let main () =
+  let v = Sys.argv.(1) in
+  let l = String.split_on_char '.' Str.(replace_first (regexp "^v") "" v) in
+  (* sanitization *)
+  let l =
+    match l with
+    | l when List.for_all is_number l -> l
+    | ["%%VERSION_NUM%%"] -> ["99";"99";"99"]
+    | _ -> Printf.eprintf "version_parser: cannot parse: %s\n" v; exit 1 in
```

Comment:
@vbgl this was my "attempt" to support development versions. I'm happy to improve on that